grpc-api/src/main/java/io/deephaven/grpc_api/session/SessionService.java

```diff
@@ -83,14 +93,19 @@ private synchronized void onFatalError(
 
     private static TerminationNotificationResponse.StackTrace transformToProtoBuf(@NotNull final Throwable throwable) {
         return TerminationNotificationResponse.StackTrace.newBuilder()
-                .setType(throwable.getClass().getSimpleName())
-                .setMessage(throwable.getMessage())
+                .setType(throwable.getClass().getName())
+                .setMessage(stringifyIfNull(throwable.getMessage()))
                 .addAllElements(Arrays.stream(throwable.getStackTrace())
+                        .limit(MAX_STACK_TRACE_DEPTH)
                         .map(StackTraceElement::toString)
                         .collect(Collectors.toList()))
                 .build();
     }
 
+    private static String stringifyIfNull(@Nullable String value) {
```

Comment:
You should delete this and just use `java.util.Objects#toString(java.lang.Object)` or `java.lang.String#valueOf(java.lang.Object)`.